connector/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -265,4 +265,11 @@ object Connect {
       .version("4.0.0")
       .bytesConf(ByteUnit.BYTE)
       .createWithDefault(1024)
+
+  val CONNECT_PROGRESS_REPORT_INTERVAL =
```

Comment:
Does it make sense to make it a runtime config?

Reply:
changed it.